spec/consensus/evidence.md

```diff
@@ -0,0 +1,199 @@
+# Evidence
+
+Evidence is an important component of Tendermint's security model. Whilst the core
+consensus protocol provides correctness gaurantees for state machine replication
+that can tolerate less than 1/3 failures, the evidence system looks to detect and
+gossip byzantine faults whose combined power is greater than  or equal to 1/3. It is worth noting that
+the evidence system is designed purely to detect possible attacks, gossip them,
+commit them on chain and inform the application running on top of Tendermint.
+Evidence in itself does not punish "bad actors", this is left to the discretion
+of the application. A common form of punishment is slashing where the validators
+that were caught violating the protocol have all or a portion of their voting
+power removed. Evidence, given the assumption that 1/3+ of the network is still
+byzantine, is susceptible to censorship and should therefore be considered added
+security on a "best effort" basis.
+
+This document walks through the various forms of evidence, how they are detected,
+gossiped, verified and committed.
+
+> NOTE: Evidence here is internal to tendermint and should not be confused with
```

Comment:
Given that this ambiguity is important enough to call out, is there a more specific name we could use throughout to make the distinction clear? E.g., "protocol evidence" or "fault evidence", in contrast with "application evidence"?

The trouble with contextually overloaded terms—particularly ones as generic as "evidence"—is that the definition doesn't follow the usage (whereas the name does).

Reply:
I find that "protocol evidence" or "fault evidence" probably wouldn't help to better distinguish whether it refers to application based evidence or Tendermint based evidence (apps also have protocols).  

Evidence is definitely a broad term. I've seen applications use the term "fraud proofs" to mean incorrect state transitions (in regards to business logic) and If I was an authoritarian dictator I would force them to keep to that term. 

Maybe we could try introduce a distinction by using terms like fault vs fraud to denote the different layers (tm and app)

Reply:
Thanks for the comment. I think naming is very important. For the scope of this specific document, I feel that replacing "evidence" with "fault evidence" will just clutter the text. At the same time, in this document we do not consider any other kinds of evidence. 

We could rename the title to `# Evidence of Validator Misbehavior`, but then stick with the term "evidence" for the rest of the document.